src/api/main/LoginController.ts

```diff
@@ -256,6 +256,10 @@ export class LoginControllerImpl implements LoginController {
 			await this.userController.deleteSession(sync)
 			this.userController = null
 			this.partialLogin = defer()
+			this.fullyLoggedIn = false
+			const locator = await this.getMainLocator()
+			locator.loginListener.reset()
```

Comment:
Should also call `reset()` in the constructor instead of initializing on the class level.